brendenwest/cascade
# Blog screen developement

## Description
Implemented async function to fetch the API data to better handle the data and errors.
Implemented the loading indicator to show while data is being fetched.
Updated images matching blog entries from the updated API
Implemented back arrow button in the innerAppBrowser instead of X
Styling changes (picture size, font size)

## To Test
1. Switch to ab-46-blogscreen-developement.
2. npm install
3. Run the app
4. Go to blog screen, check if loading indicator is there, if links are working, etc.
